src/test/java/core/basesyntax/service/impl/SaverDataToStorageImplTest.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.dto.FruitRecordDto;
+import core.basesyntax.service.ApplierFruitsToStorage;
+import core.basesyntax.storage.DataBase;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class SaverDataToStorageImplTest {
+    private static final AddHandlerImpl addHandler = new AddHandlerImpl();
+    private static final SaverDataToStorageImpl saverDataToStorage
+            = new SaverDataToStorageImpl();
+    private static final ApplierFruitsToStorage purchaseHandler
+            = new PurchaseFruitHandlerImpl();
+    private static Map<String, Integer> testDataBase;
+    private static OperationType operationType;
+    private static FruitRecordDto firstFruitRecordDto;
+    private static FruitRecordDto secondFruitRecordDto;
+    private static int firstFruitDtoAmount;
+    private static int secondFruitDtoAmount;
+    private static List<FruitRecordDto> fruitDtos;
+    private static Map<OperationType, ApplierFruitsToStorage> operationStrategyMap;
+
+    @BeforeEach
+    void setUp() {
+        testDataBase = new HashMap<>();
+        operationType = OperationType.SUPPLY;
+        firstFruitDtoAmount = 30;
+        secondFruitDtoAmount = 50;
+        firstFruitRecordDto = new FruitRecordDto(operationType, "apple", firstFruitDtoAmount);
+        secondFruitRecordDto = new FruitRecordDto(OperationType
+                .SUPPLY, "apple", secondFruitDtoAmount);
+        fruitDtos = new ArrayList<>();
+        operationStrategyMap
+                = new HashMap<>();
+    }
+
+    @AfterEach
+    void tearDown() {
```

Comment:
If you have single test, it doesn't look like you need AfterEach

Reply:
Fixed